### ISSUE TITLE
No incremental backups for several hours, configured to 15 minutes

### ISSUE THREAD

**Issue by peter-ahe** [ChaoticTrials/SimpleBackups#19]
### Minecraft version

1.19.2

### Simple Backups version

1.19.1-2.1.9

### Forge version

43.2.14

### The latest.log file

https://gist.github.com/peter-ahe/2fc712ee43b95fd3d7922e614f748e79

### Issue description

I noticed there were no new backups for several hours.

I don't know if this is a duplicate of #14 as I had some directories in the backup directory? I've moved them elsewhere now, so they're not included in the listing below.

This is a list of backups:

```
simplebackups/New World_2023-08-15_11-31-49.zip
simplebackups/New World_2023-08-15_09-01-49.zip
simplebackups/New World_2023-08-15_04-16-47.zip
simplebackups/New World_2023-08-15_05-31-48.zip
simplebackups/New World_2023-08-15_09-46-49.zip
simplebackups/New World_2023-08-15_10-01-49.zip
simplebackups/New World_2023-08-15_08-31-48.zip
simplebackups/New World_2023-08-15_10-46-49.zip
simplebackups/New World_2023-08-15_12-18-02.zip
simplebackups/New World_2023-08-15_14-18-05.zip
simplebackups/New World_2023-08-15_02-16-43.zip
simplebackups/New World_2023-08-15_01-01-43.zip
simplebackups/New World_2023-08-14_23-16-41.zip
simplebackups/New World_2023-08-15_07-16-48.zip
simplebackups/New World_2023-08-15_01-46-43.zip
simplebackups/New World_2023-08-15_00-31-41.zip
simplebackups/New World_2023-08-15_15-18-07.zip
simplebackups/New World_2023-08-15_08-46-48.zip
simplebackups/New World_2023-08-15_10-31-49.zip
simplebackups/New World_2023-08-15_08-01-48.zip
simplebackups/New World_2023-08-15_13-18-03.zip
simplebackups/New World_2023-08-15_09-31-49.zip
simplebackups/New World_2023-08-15_11-01-49.zip
simplebackups/New World_2023-08-15_05-46-48.zip
simplebackups/New World_2023-08-15_15-48-08.zip
simplebackups/New World_2023-08-15_11-46-49.zip
simplebackups/New World_2023-08-15_00-46-42.zip
simplebackups/New World_2023-08-15_06-16-48.zip
simplebackups/New World_2023-08-15_12-01-50.zip
simplebackups/New World_2023-08-15_16-18-08.zip
simplebackups/New World_2023-08-14_22-16-40.zip
simplebackups/New World_2023-08-14_21-40-26.zip
simplebackups/New World_2023-08-15_03-16-44.zip
simplebackups/New World_2023-08-15_01-31-43.zip
simplebackups/New World_2023-08-15_00-01-41.zip
simplebackups/New World_2023-08-14_23-31-41.zip
simplebackups/New World_2023-08-15_07-31-48.zip
simplebackups/New World_2023-08-15_03-01-44.zip
simplebackups/New World_2023-08-15_00-16-41.zip
simplebackups/New World_2023-08-15_13-48-05.zip
simplebackups/New World_2023-08-15_06-01-48.zip
simplebackups/New World_2023-08-15_03-46-46.zip
simplebackups/New World_2023-08-15_06-46-48.zip
simplebackups/New World_2023-08-14_22-46-41.zip
simplebackups/New World_2023-08-15_16-03-08.zip
simplebackups/New World_2023-08-14_22-01-40.zip
simplebackups/New World_2023-08-15_16-48-09.zip
simplebackups/New World_2023-08-15_13-03-03.zip
simplebackups/New World_2023-08-15_11-16-49.zip
simplebackups/New World_2023-08-15_14-33-06.zip
simplebackups/New World_2023-08-15_05-16-48.zip
simplebackups/New World_2023-08-15_04-31-47.zip
simplebackups/New World_2023-08-15_12-33-02.zip
simplebackups/New World_2023-08-15_02-31-43.zip
simplebackups/New World_2023-08-15_05-01-47.zip
simplebackups/New World_2023-08-15_15-03-06.zip
simplebackups/New World_2023-08-15_08-16-48.zip
simplebackups/New World_2023-08-15_16-33-09.zip
simplebackups/New World_2023-08-15_06-31-48.zip
simplebackups/New World_2023-08-14_22-31-41.zip
simplebackups/New World_2023-08-15_01-16-43.zip
simplebackups/New World_2023-08-14_23-01-41.zip
simplebackups/New World_2023-08-15_07-01-48.zip
simplebackups/New World_2023-08-15_13-33-04.zip
simplebackups/New World_2023-08-14_23-46-41.zip
simplebackups/New World_2023-08-15_07-46-48.zip
simplebackups/New World_2023-08-15_03-31-45.zip
simplebackups/New World_2023-08-15_10-16-49.zip
simplebackups/New World_2023-08-15_15-33-07.zip
simplebackups/New World_2023-08-15_02-46-43.zip
simplebackups/New World_2023-08-15_14-03-05.zip
simplebackups/New World_2023-08-15_12-48-03.zip
simplebackups/New World_2023-08-15_02-01-43.zip
simplebackups/New World_2023-08-15_04-46-47.zip
simplebackups/New World_2023-08-15_09-16-49.zip
simplebackups/New World_2023-08-15_14-48-06.zip
simplebackups/New World_2023-08-15_04-01-47.zip
```

This is my config:

```toml
#If set false, no backups are being made.
enabled = true
#Should only changed files be backed up? Useful for large worlds. Keep in mind that old backups are required for a complete backup. Alternatively, you could run the command to create a new complete backup.
onlyModified = true
#How often should a full backup be created if only modified files should be saved? This creates a full backup when x minutes are over and the next backup needs to be done. Once a year is default.
#Range: 1 ~ 5259600
fullBackupTimer = 525960
#The max amount of backup files to keep.
#Range: 1 ~ 32767
backupsToKeep = 80
#The time between two backups in minutes
#5 = each 5 minutes
#60 = each hour
#1440 = each day
#Range: 1 ~ 32767
timer = 15
#The compression level, 0 is no compression (less cpu usage) and takes a lot of space, 9 is best compression (most cpu usage) and takes less space. -1 is default
#Range: -1 ~ 9
compressionLevel = -1
#Should message be sent when backup is in the making?
sendMessages = true
#The max size of storage the backup folder. If it takes more storage, old files will be deleted.
#Needs to be written as <number><space><storage type>
#Valid storage types: B, KB, MB, GB, TB
maxDiskSize = "50 GB"
#Used to define the output path.
outputPath = "simplebackups"

[mod_compat]
	#Should backup notifications be sent to Discord by using mc2discord? (needs to be installed)
	mc2discord = true


```

### Steps to reproduce

Unfortunately, I don't know how.

### Other information

I'm playing All The Mods 8-1.0.22, with this list of mods:

I did have to tweak a JSON file in Apotheosis. I did that by extracting the jar file to a directory, so `mods/Apotheosis-1.19.2-6.2.1.jar` is actually a directory.

```
mods/item-filters-forge-1902.2.9-build.51.jar
mods/DimStorage-1.19.2-7.2.0.jar
mods/allthemodium-2.1.6-1.19.2-43.1.1.jar
mods/botarium-forge-1.19.2-1.8.2.jar
mods/paucal-forge-1.19.2-0.5.0.jar
mods/AutoRegLib-1.8.2-55.jar
mods/DefaultSettings-1.19.2-4.0.5-Forge.jar
mods/rsrequestify-2.3.0.jar
mods/Apotheosis-1.19.2-6.2.1.jar
mods/voidtotem-forge-1.19.2-2.1.0.jar
mods/curios-forge-1.19.2-5.1.4.1.jar
mods/IronJetpacks-1.19.2-6.0.4.jar
mods/morered-1.19.2-3.0.0.1.jar
mods/modnametooltip-1.19-1.19.0.jar
mods/ars_nouveau-1.19.2-3.17.6.jar
mods/thermal_cultivation-1.19.2-10.2.0.17.jar
mods/Controlling-forge-1.19.2-10.0+7.jar
mods/cabletiers-1.19.2-0.5471.jar
mods/ftb-quests-forge-1902.4.18-build.244.jar
mods/AdvancedPeripherals-1.19.2-0.7.29r.jar
mods/YungsBetterDungeons-1.19.2-Forge-3.2.2.jar
mods/ad_astra-forge-1.19.2-1.12.6.jar
mods/NaturesAura-38.2.jar
mods/MyServerIsCompatible-1.19-1.0.jar
mods/ensorcellation-1.19.2-4.2.0.14.jar
mods/TravelAnchors-1.19.2-4.1.2.jar
mods/tetra-1.19.2-5.3.1.jar
mods/kubejs-forge-1902.6.0-build.142.jar
mods/tetranomicon-1.4.2-1.19.2.jar
mods/thermal_integration-1.19.2-10.2.0.17.jar
mods/ftb-essentials-1902.1.10-build.47.jar
mods/modonomicon-1.19.2-1.33.0.jar
mods/phosphophyllite-1.19.2-0.6.0-beta.7.1.jar
mods/villagertools-1.19-1.0.3.jar
mods/creeperoverhaul-2.0.9-forge.jar
mods/FramedBlocks-6.8.4.jar
mods/entityculling-forge-1.6.1-mc1.19.2.jar
mods/modular-routers-1.19.2-10.2.0-3.jar
mods/inventoryessentials-forge-1.19-5.0.2.jar
mods/ExtraDisks-1.19.2-2.2.0.jar
mods/letmedespawn-1.18.x-1.19.x-forge-1.0.3.jar
mods/absentbydesign-1.19-1.7.0.jar
mods/thermal_expansion-1.19.2-10.2.0.21.jar
mods/AIOTBotania-1.19.2-3.0.1.jar
mods/flickerfix-1.19.1-3.1.0.jar
mods/resourcefullib-forge-1.19.2-1.1.24.jar
mods/ThermalExtra 1.19.2-3.0.3.jar
mods/supplementaries-1.19.2-2.3.20.jar
mods/additional_lights-1.19-2.1.6.jar
mods/ToolBelt-1.19.2-1.19.7.jar
mods/geckolib-forge-1.19-3.1.40.jar
mods/SuperFactoryManager-1.19.2-4.10.3.jar
mods/farsight-1.19.2-2.1.jar
mods/twilightforest-1.19.2-4.2.1518-universal.jar
mods/Oh_The_Biomes_You'll_Go-forge-1.19.2-2.0.1.4.jar
mods/MekanismGenerators-1.19.2-10.3.8.477.jar
mods/.DS_Store
mods/rangedpumps-1.0.0.jar
mods/RootsClassic-1.19.2-1.1.35.jar
mods/sophisticatedstorage-1.19.2-0.8.34.523.jar
mods/ftb-industrial-contraptions-1900.1.7-build.212.jar
mods/harvestwithease-1.19.2-6.0.1.4-final-forge.jar
mods/HostileNeuralNetworks-1.19.2-4.1.0.jar
mods/LibX-1.19.2-4.2.8.jar
mods/crashutilities-6.2.jar
mods/AEAdditions-1.19.2-4.0.3.jar
mods/JCPlugin-1.17.x-1.20.x-4.0.4-Forge.jar
mods/enderchests-forge-1.19.0-1.10.1.03.jar
mods/YungsBetterMineshafts-1.19.2-Forge-3.2.0.jar
mods/almostunified-forge-1.19.2-0.5.0.jar
mods/HealthOverlay-1.19.2-7.2.1.jar
mods/elytraslot-forge-6.1.1+1.19.2.jar
mods/SandBox-1.19.2-1.0.1.jar
mods/FarmersDelight-1.19-1.2.1.jar
mods/refinedstorageaddons-0.9.0.jar
mods/naturalist-forge-3.0.3a-1.19.2.jar
mods/InventoryProfilesNext-forge-1.19-1.10.6.jar
mods/blueflame-1.19.2-0.1.0.2.jar
mods/repurposed_structures_forge-6.3.24+1.19.2.jar
mods/shetiphiancore-forge-1.19.0-3.11.3.04.jar
mods/mutil-1.19.2-5.1.0.jar
mods/cofh_core-1.19.2-10.2.1.40.jar
mods/Powah-4.0.10.jar
mods/simplemagnets-1.1.9-forge-mc1.19.jar
mods/SimpleBackups-1.19.1-2.1.9.jar
mods/FastWorkbench-1.19.2-7.1.3.jar
mods/ScalableCatsForce-2.13.10-build-10-with-library.jar
mods/minecolonies-1.19.2-1.0.1489-BETA.jar
mods/AE2-Things-1.1.1.jar
mods/MekanismTools-1.19.2-10.3.8.477.jar
mods/configured-2.1.1-1.19.2.jar
mods/journeymap-1.19.2-5.9.7-forge.jar
mods/DungeonsArise-1.19.2-2.1.56-beta.jar
mods/titanium-1.19.2-3.7.3-27.jar
mods/theoneprobe-1.19-6.2.2.jar
mods/ftb-ultimine-forge-1902.4.1-build.90.jar
mods/createaddition-1.19.2-20230623a.jar
mods/oculus-mc1.19.2-1.6.4.jar
mods/constructionwand-1.19.2-2.10.jar
mods/ftb-chunks-forge-1902.3.22-build.271.jar
mods/MysticalAgriculture-1.19.2-6.0.10.jar
mods/easy_villagers-1.19.2-1.0.11.jar
mods/engineersdecor-1.19.2-forge-1.3.28.jar
mods/AEInfinityBooster-1.19.2-1.2.0+11.jar
mods/MythicBotany-1.19.2-3.0.2.jar
mods/thermal_foundation-1.19.2-10.2.0.47.jar
mods/corail_woodcutter-1.19.2-2.5.2.jar
mods/IntegratedTunnels-1.19.2-1.8.20.jar
mods/BotanyPots-Forge-1.19.2-9.0.33.jar
mods/rftoolsutility-1.19-5.1.7.jar
mods/dankstorage-1.19.2-5.1.6.jar
mods/libIPN-forge-1.19-3.0.2.jar
mods/additionallanterns-1.0.5-forge-mc1.19.jar
mods/AE2WTLib-12.9.5.jar
mods/compactmachines-5.1.0.jar
mods/blue_skies-1.19.2-1.3.20.jar
mods/JustEnoughResources-1.19.2-1.2.2.200.jar
mods/crafting-on-a-stick-1.19.2-1.0.5.jar
mods/rftoolsbuilder-1.19-5.2.8.jar
mods/alltheores-2.0.2-1.19.2-43.1.3.jar
mods/Iceberg-1.19.2-forge-1.1.4.jar
mods/Aquaculture-1.19.2-2.4.8.jar
mods/SmartBrainLib-forge-1.19.2-1.9.jar
mods/appleskin-forge-mc1.19-2.4.2.jar
mods/waystones-forge-1.19.2-11.4.0.jar
mods/structure_gel-1.19.2-2.7.2.jar
mods/getittogetherdrops-forge-1.19.2-1.3.jar
mods/mob_grinding_utils-1.19.2-0.4.50.jar
mods/EvilCraft-1.19.2-1.2.24.jar
mods/MysticalCustomization-1.19.2-4.0.1.jar
mods/YungsApi-1.19.2-Forge-3.8.9.jar
mods/YungsBetterOceanMonuments-1.19.2-Forge-2.1.0.jar
mods/occultism-1.19.2-1.81.0.jar
mods/littlelogistics-mc1.19.2-v1.4.0.jar
mods/chemlib-1.19.2-2.0.17.jar
mods/farmingforblockheads-forge-1.19.2-11.3.0.jar
mods/Measurements-forge-1.19.2-1.3.1.jar
mods/Shrink-1.19-1.3.5.jar
mods/Prism-1.19.1-1.0.2.jar
mods/rebornstorage-1.19.2-5.0.3.jar
mods/time-in-a-bottle-3.0.1-mc1.19.jar
mods/spirit-forge-1.19.2-2.2.6.jar
mods/dpanvil-1.19.2-4.4.0.jar
mods/redstone_arsenal-1.19.2-7.2.0.15.jar
mods/AdditionalEnchantedMiner-1.19.2-1192.2.0.jar
mods/mcjtylib-1.19-7.2.5.jar
mods/caelus-forge-1.19.2-3.0.0.6.jar
mods/FluxNetworks-1.19.2-7.1.4.13.jar
mods/findme-3.1.0-forge.jar
mods/Morph-o-Tool-1.6-34.jar
mods/modelfix-1.8.jar
mods/refinedcooking-3.0.3.jar
mods/BetterAdvancements-1.19.2-0.2.2.142.jar
mods/thermal_locomotion-1.19.2-10.2.0.14.jar
mods/productivebees-1.19.2-0.10.7.2.jar
mods/Steam_Rails-1.3.5+forge-mc1.19.2.jar
mods/XyCraft Override-0.5.17.jar
mods/blockui-1.19-0.0.69-ALPHA.jar
mods/saturn-mc1.19.2-0.0.4.jar
mods/CorgiLib-forge-1.19.2-1.0.0.34.jar
mods/lazydfu-1.19-1.0.2.jar
mods/moredragoneggs-3.2.jar
mods/MysticalAgradditions-1.19.2-6.0.3.jar
mods/itemcollectors-1.1.8-forge-mc1.19.jar
mods/ExperienceBugFix-1.19-1.41.2.3.jar
mods/DungeonCrawl-1.19-2.3.12.jar
mods/Botania-1.19.2-440-FORGE.jar
mods/Applied-Botanics-1.4.2.jar
mods/lootr-1.19-0.4.24.62.jar
mods/ponderjs-1.19.2-1.1.12.jar
mods/PackMenu-1.19.2-5.1.0.jar
mods/GameMenuModOption-1.19-1.18.jar
mods/craftingtweaks-forge-1.19.2-15.1.7.jar
mods/trashslot-forge-1.19-12.0.2.jar
mods/Croptopia-1.19.2-FORGE-2.2.2.jar
mods/ftb-library-forge-1902.3.19-build.214.jar
mods/hexal-forge-1.19.2-0.2.18.jar
mods/reliquary-1.19.2-2.0.20.1166.jar
mods/deepresonance-1.19-4.1.3.jar
mods/RSInfinityBooster-1.19.2-3.0+27.jar
mods/ferritecore-5.0.3-forge.jar
mods/ExtraStorage-1.19.2-3.0.1.jar
mods/tetrasdelight-1.19.2-1.jar
mods/elevatorid-1.19.2-1.8.9.jar
mods/pneumaticcraft-repressurized-1.19.2-4.3.5-25.jar
mods/architectury-6.5.85-forge.jar
mods/GravitationalModulatingAdditionalUnit-1.19.2-2.8.jar
mods/mininggadgets-1.13.0.jar
mods/IntegratedDynamics-1.19.2-1.16.8.jar
mods/pipez-1.19.2-1.1.3.jar
mods/moreoverlays-1.21.9-mc1.19.2.jar
mods/UtilitiX-1.19.2-0.7.11.jar
mods/rubidium-0.6.2b.jar
mods/StructureCompass-1.19.2-1.4.3.jar
mods/kotlinforforge-3.12.0-all.jar
mods/pylons-1.19.2-3.1.0.jar
mods/connectivity-1.19.2-4.4.jar
mods/starlight-1.1.1+forge.cf5b10b.jar
mods/rsgauges-1.19.2-forge-1.2.19.jar
mods/bdlib-1.25.0.5-mc1.19.2.jar
mods/balm-forge-1.19.2-4.5.7.jar
mods/CommonCapabilities-1.19.2-2.9.0.jar
mods/advgenerators-1.4.0.5-mc1.19.2.jar
mods/Patchouli-1.19.2-77.jar
mods/connectedglass-1.1.7-forge-mc1.19.jar
mods/Runelic-Forge-1.19.2-14.1.4.jar
mods/littlecontraptions-forge-1.19.2.2.jar
mods/trashcans-1.0.18-forge-mc1.19.jar
mods/integratednbt-1.19.2-1.6.0.jar
mods/myrtrees-forge-1.2.0-build.32.jar
mods/MaxHealthFix-Forge-1.19.2-8.0.1.jar
mods/handcrafted-forge-1.19.2-2.0.6.jar
mods/async-locator-forge-1.19.2-1.3.0.jar
mods/appliedcooking-2.0.3.jar
mods/MobDespawnTimers-1.19.2-2.0.1.jar
mods/MouseTweaks-forge-mc1.19-2.23.jar
mods/DarkModeEverywhere-1.19.2-1.1.3.jar
mods/mcw-doors-1.0.9forge-mc1.19.2.jar
mods/nomowanderer-1.19.2_1.3.11.jar
mods/elementalcraft-1.19.2-5.8.5.jar
mods/yeetusexperimentus-1.0.1-build.2+mc1.19.1.jar
mods/Entity_Collision_FPS_Fix-forge-1.19-2.0.0.0.jar
mods/domum_ornamentum-1.19-1.0.76-ALPHA-universal.jar
mods/wirelesschargers-1.0.9-forge-mc1.19.2.jar
mods/multipiston-1.19.2-1.2.21-ALPHA.jar
mods/theurgy-1.19.2-1.3.5.jar
mods/simplylight-1.19.2-1.4.5-build.42.jar
mods/EdivadLib-1.19.2-1.2.0.jar
mods/modernfix-forge-5.2.0+mc1.19.2.jar
mods/JustEnoughProfessions-forge-1.19.2-2.0.2.jar
mods/antighost-1.19.1-forge42.0.1-1.1.3.jar
mods/Placebo-1.19.2-7.3.0.jar
mods/mcw-roofs-2.2.4-mc1.19.2forge.jar
mods/extremesoundmuffler-3.37-forge-1.19.2.jar
mods/silent-lib-1.19.2-7.0.3.jar
mods/industrial-foregoing-1.19.2-3.3.2.3-5.jar
mods/SpitItOut-1.19.4-1.0.0.jar
mods/thermal_innovation-1.19.2-10.2.0.18.jar
mods/moonlight-1.19.2-2.2.43-forge.jar
mods/FastSuite-1.19.2-4.1.0.jar
mods/refinedstorage-1.11.6.jar
mods/supermartijn642configlib-1.1.7-forge-mc1.19.jar
mods/BetterF3-4.0.0-Forge-1.19.2.jar
mods/platforms-forge-1.19.0-1.10.2.02.jar
mods/YungsBetterDesertTemples-1.19.2-Forge-2.2.2.jar
mods/rftoolsstorage-1.19-4.1.2.jar
mods/mcw-fences-1.0.7-mc1.19.2forge.jar
mods/ReAuth-1.19-Forge-4.0.7.jar
mods/netherportalfix-forge-1.19-10.0.1.jar
mods/silent-gear-1.19.2-3.2.6.jar
mods/Quark-3.4-405.jar
mods/glassential-forge-1.19-1.2.4.jar
mods/alchemylib-1.19.2-1.0.21.jar
mods/eccentrictome-1.19.2-1.10.1.jar
mods/quartz-1.19.2-0.1.0-beta.2.1.jar
mods/Ad-Astra-Giselle-Addon-forge-1.19.2-1.16.jar
mods/chipped-forge-1.19.2-2.1.5.jar
mods/supermartijn642corelib-1.1.10-forge-mc1.19.2.jar
mods/Delightful-1.19.2-3.3.jar
mods/FastFurnace-1.19.2-7.0.0.jar
mods/AllTheCompressed-1.19.2-2.0.0.jar
mods/jei-1.19.2-forge-11.6.0.1016.jar
mods/BotanyTrees-Forge-1.19.2-5.0.8.jar
mods/cloth-config-8.2.88-forge.jar
mods/rftoolsbase-1.19-4.1.7.jar
mods/tesseract-1.0.35-forge-mc1.19.jar
mods/XyCraft Core-0.5.17.jar
mods/Applied-Mekanistics-1.3.4.jar
mods/rftoolscontrol-1.19-6.1.4.jar
mods/artifacts-1.19.2-5.0.3.jar
mods/potionsmaster-0.6.0-1.19.2-43.1.1.jar
mods/PigPen-Forge-1.19.2-11.1.2.jar
mods/adtetra-1.0.1.jar
mods/hexcasting-forge-1.19.2-0.10.3.jar
mods/DarkUtilities-Forge-1.19.2-13.1.9.jar
mods/rhino-forge-1902.2.2-build.268.jar
mods/allthetweaks-2.0.4-1.19.2-43.1.3.jar
mods/deeperdarker-forge-1.1.6-forge.jar
mods/AngelRing2-1.19.2-2.1.5.jar
mods/systeams-1.19.2-1.4.0.jar
mods/fuelgoeshere-1.19.2-0.1.0.0.jar
mods/ars_creo-1.19.2-3.2.1.jar
mods/jmi-forge-1.19.2-0.13-33.jar
mods/Cucumber-1.19.2-6.0.7.jar
mods/YungsBetterStrongholds-1.19.2-Forge-3.2.0.jar
mods/cookingforblockheads-forge-1.19.2-13.3.2.jar
mods/TerraBlender-forge-1.19.2-2.0.1.136.jar
mods/hexerei-0.3.2.2.jar
mods/EnchantmentDescriptions-Forge-1.19.2-13.0.14.jar
mods/structurize-1.19.2-1.0.492-ALPHA.jar
mods/SoL-Carrot-1.19.2-1.14.0.jar
mods/merequester-1.19.2-1.1.2.jar
mods/fusion-1.0.3-forge-mc1.19.2.jar
mods/Mekanism-1.19.2-10.3.8.477.jar
mods/charmofundying-forge-6.2.1+1.19.2.jar
mods/ImmersiveEngineering-1.19.2-9.2.2-165.jar
mods/mahoutsukai-1.19.2-v1.34.52.jar
mods/ftb-ranks-forge-1902.1.15-build.77.jar
mods/ctov-3.2.5.jar
mods/polymorph-forge-0.46.4+1.19.2.jar
mods/biggerreactors-1.19.2-0.6.0-beta.6.1.jar
mods/rechiseled-1.1.1-forge-mc1.19.2.jar
mods/arsarmiger-1.19.2-1.4.0.jar
mods/functionalstorage-1.19.2-1.1.3.jar
mods/baubley-heart-canisters-1.19.2-2.1.0.jar
mods/BambooEverything-forge-2.2.4-build.33+mc1.19.2.jar
mods/create-1.19.2-0.5.1.c.jar
mods/FastLeafDecay-30.jar
mods/CyclopsCore-1.19.2-1.18.6.jar
mods/mcw-trapdoors-1.1.1-mc1.19.2forge.jar
mods/dummmmmmy-1.19.2-1.7.1.jar
mods/buildinggadgets-3.16.2-build.22+mc1.19.2.jar
mods/NoChatReports-FORGE-1.19.2-v1.5.1.jar
mods/charginggadgets-1.9.0.jar
mods/vitalize-forge-1.19.2-1.1.1.jar
mods/ironfurnaces-1.19.2-3.6.5.jar
mods/AdvancementPlaques-1.19.2-1.4.7.jar
mods/xnet-1.19-5.1.4.jar
mods/alchemistry-1.19.2-2.3.0.jar
mods/thermal_dynamics-1.19.2-10.2.1b.14.jar
mods/spark-1.10.37-forge.jar
mods/cc-tweaked-1.19.2-1.101.3.jar
mods/megacells-forge-2.0.0-1.19.2.jar
mods/[1.19.2] SecurityCraft v1.9.6.1.jar
mods/bwncr-forge-1.19.2-3.14.1.jar
mods/cfm-7.0.0-pre35-1.19.2.jar
mods/rftoolspower-1.19-5.1.2.jar
mods/torchmaster-19.2.90.jar
mods/laserio-1.5.2.jar
mods/mcw-bridges-2.1.0-mc1.19.2forge.jar
mods/YungsBetterWitchHuts-1.19.2-Forge-2.1.0.jar
mods/Bookshelf-Forge-1.19.2-16.3.20.jar
mods/CosmeticArmorReworked-1.19.2-v1a.jar
mods/jumpboat-1.19-0.1.0.5.jar
mods/sebastrnlib-2.0.2.jar
mods/sophisticatedbackpacks-1.19.2-3.18.53.868.jar
mods/tombstone-8.2.10-1.19.2.jar
mods/comforts-forge-6.0.5+1.19.2.jar
mods/Clumps-forge-1.19.2-9.0.0+14.jar
mods/ToastControl-1.19.2-7.0.0.jar
mods/GatewaysToEternity-1.19.2-3.2.0.jar
mods/WitherSkeletonTweaks-1.19.2-8.0.4.jar
mods/appliedenergistics2-forge-12.9.5.jar
mods/xnetgases-1.19.2-4.0.1.jar
mods/XyCraft World-0.5.17.jar
mods/NaturesCompass-1.19.2-1.10.0-forge.jar
mods/ftb-teams-forge-1902.2.14-build.123.jar
mods/sophisticatedcore-1.19.2-0.5.78.358.jar
mods/energymeter-1.19.2-1.0.0.jar
mods/ars_elemental-1.19.2-0.5.9.1.2.jar
mods/tempad-forge-1.19.2-1.4.5.jar
mods/universalgrid-1.19.2-1.1.jar
mods/clickadv-1.19.2-3.0.jar
mods/endertanks-forge-1.19.0-1.12.1.04.jar
mods/AttributeFix-Forge-1.19.2-17.2.6.jar
mods/entangled-1.3.14-forge-mc1.19.2.jar
mods/observable-3.3.1.jar
mods/YungsExtras-1.19.2-Forge-3.1.0.jar
mods/mcw-lights-1.0.6-mc1.19.2forge.jar
mods/IntegratedTerminals-1.19.2-1.4.7.jar
mods/LegendaryTooltips-1.19.2-forge-1.4.0.jar
mods/IntegratedCrafting-1.19.2-1.1.2.jar
mods/resourcefulconfig-forge-1.19.2-1.0.20.jar
mods/CTM-1.19.2-1.1.6+8.jar
```

**Comment by peter-ahe** [comment]
Also, I opened to LAN and tried initiating a manual backup. Nothing happened.

**Comment by MelanX** [comment]
1. Was anybody online when the backups were skipped? 
2. Were other backups running when a new backup should start? 
3. Did it work after restarting?
4. Were the backups being created when you already had sub-directories in the backup output path? 
5. Did you try removing the sub-directories, and run the command?

I never checked the mod with `Open to LAN`, should do that.

**Comment by peter-ahe** [comment]
Thank you for your quick response.

> 1. Was anybody online when the backups were skipped?

Single player, but AFK.

> 2. Were other backups running when a new backup should start?

I don't know how to see if a backup was running.

> 3. Did it work after restarting?

It appears to be working now, after restart and removing the subdirectories.

> 4. Were the backups being created when you already had sub-directories in the backup output path?

Yes. But I don't think it had started deleting. The number of backups listed above is 77 + 2 subdirectories. Which is pretty close to 80 when it should start deleting. After seeing #14, my guess is that it entered the infinite loop.

> 5. Did you try removing the sub-directories, and run the command?

The command works now, after restarting and removing subdirectories.

**Comment by MelanX** [comment]
I think it's a duplicate of #14 as you mentioned before. I uploaded a fix for 1.19.2 for this bug now on [CurseForge](https://legacy.curseforge.com/minecraft/mc-mods/simple-backups/files/4720393) and [Modrinth](https://modrinth.com/mod/simple-backups/version/1.19.1-2.1.10).
